userguide/storage.rst

```diff
@@ -1701,12 +1701,10 @@ First, make sure that a passphrase has been set using the instructions
 in :ref:`Encryption` **before** attempting to replace the failed
 drive. Then, follow the steps 1 and 2 as described above. During step
 3, a prompt will appear to input and confirm the passphrase for the
-pool. Enter this information then click the :guilabel:`Replace Disk`
-button. Wait until the resilvering is complete.
-
-Next, restore the encryption keys to the pool.
-**If this additional step is not performed before the next
-reboot, access to the pool might be permanently lost.**
+pool. Enter this information then click :guilabel:`Replace Disk`.
+Immediately restore the encryption keys to the pool. **If this
+additional step is not performed before the next reboot, access to the
```

Comment:
Suggestion to make this warning a little more specific and stronger:
**WARNING: Access to the pool will be permanently lost unless the encryption keys are restored to the pool before the next system reboot!**